Eat_and_Greet/src/screens/ProfileScreen/index.tsx

```diff
@@ -1,20 +1,172 @@
-import { Text, View } from 'react-native';
+import React, {useState} from 'react';
+import { Text, View, TouchableOpacity, ScrollView, StyleSheet } from 'react-native';
 import { StatusBar } from 'expo-status-bar';
 
 import { Toolbar } from '../../comps/Toolbar/toolbar';
-
+import ChooseImageButton from "../../comps/ChooseImageButton/ChooseImageButton";
 import styles from '../../style';
+import SplitButton from '../../comps/SplitButtons/SplitButton';
+import UserInformation from '../../comps/UserInformation/UserInformation';
 
 function ProfileScreen() {
+  const [UserInfoVisible, setUserInfoVisible] = useState(true);
+  const [EventsHostedVisible, setEventsHostedVisible] = useState(false);
+  const [EventsAttendedVisible, setEventsAttendedVisible] = useState(false);
```

Comment:
Not at all necessary, but could perhaps use a single state of SectionVisible with 0 being UserInfo, 1 being EventsHosted, & 2 being EventsAttended. Would make code cleaner but maybe a little harder to understand